pkg/models/plan.go

```diff
@@ -1,32 +1,46 @@
 package models
 
 import (
-	"github.com/bacalhau-project/bacalhau/pkg/jobstore"
 	"github.com/bacalhau-project/bacalhau/pkg/model"
 )
 
+type PlanExecutionDesiredUpdate struct {
+	Execution    *model.ExecutionState
+	DesiredState model.ExecutionDesiredState
+	Comment      string
+}
+
 // Plan holds actions as a result of processing an evaluation by the scheduler.
 type Plan struct {
 	EvalID      string
 	EvalReceipt string
-	Priority    int
+	// TODO: passing the evalID should be enough once we persist evaluations
+	Eval     *Evaluation
+	Priority int
+
+	Job        *model.Job
+	JobVersion int
+
+	DesiredJobState model.JobStateType
+	Comment         string
 
 	// NewExecutions holds the executions to be created.
 	NewExecutions []*model.ExecutionState
 
-	// StoppedExecutions holds the executions to be stopped.
-	StoppedExecutions []*jobstore.UpdateExecutionRequest
+	UpdatedExecutions map[model.ExecutionID]*PlanExecutionDesiredUpdate
```

Comment:
maybe call this `executionUpdates` or `activeExecutions`? I think it represents a map of execution states previously in `NewExecutions` that are now pending or running?

Reply:
can't call them `activeExecutions` since they include  updating the state of executions to be marked as failed or cancelled.